### PR TITLE
Fixed Image search from image description Page 

### LINKED ISSUE #616: Image search from image description won't work with return key
## Describe the bug
When we search and open an image details , and use the top search widget to search pressing the return key to search only places the text in search bar and returns to the search section

## To Reproduce
1. Go to 'https://search.creativecommons.org/search?license=&license_type=&categories=&extension=&aspect_ratio=&size=&source=&q=avril&searchBy='
2. Click on any image 
3. Use the search widget type the query and press return
4. It will redirect to previous search results but with the new query on searchbox

## Expected behavior
When search is performed using the return key in the search widget it should return the actual query

## Screenshots
image

### Type the query

image

### Hit return key

image

## Desktop (please complete the following information)
 - OS: Windows 10
 - Browser Chrome
 - Version  79.0.3945.130

### PR DESCRIPTION
**Pull request by abhisheknaiidu**
Fixes #616 

[Short description explaining the high-level reason for the pull request]

Now the Search Bar of Nav-Bar Works fine.

Developer Certificate of Origin Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors. 1 Letterman Drive Suite D4700 San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

### CODE REVIEW

File: src/components/SocialShareButtons.vue [hunk 1]
```diff
@@ -17,7 +17,7 @@
         class="social-button pinterest"
         target="_blank"
         @click="onSocialMediaLinkClick('Pinterest')"
-        :href="`https://www.pinterest.com/pin/create/bookmarklet/?media=${imageURL}&description=${shareText}`">
+        :href="`https://www.pinterest.com/pin/create/bookmarklet/?media=${imageURL}&is_video=false&description=${shareText}`">
```
Comment (akmadian) [inline src/components/SocialShareButtons.vue]: These changes are causing unit testing to fail. Please address this, here's the error info:
```
Summary of all failing tests
 FAIL  test/unit/specs/components/social-share-buttons.spec.js (5.23s)
  ● SocialShareButtons › should render all buttons with correct query params

    expect(string).toContain(value)
    
    Expected string:
      "<a target=\"_blank\" href=\"https://www.pinterest.com/pin/create/bookmarklet/?media=http://image.url&amp;is_video=false&amp;description=share text\" class=\"social-button pinterest\"></a>"
    To contain value:
      "?media=http://image.url&amp;description=share text"

      35 |     expect(wrapper.find('.social-button.facebook').html()).toContain('?u=http://share.url&amp;t==share');
      36 |     expect(wrapper.find('.social-button.twitter').html()).toContain('?text=share text');
    > 37 |     expect(wrapper.find('.social-button.pinterest').html()).toContain('?media=http://image.url&amp;description=share text');
      38 |   });
      39 | 
      40 |   it('dispatches social media share event when facebook link is clicked', () => {
      
      at Object.<anonymous> (test/unit/specs/components/social-share-buttons.spec.js:37:61)
```

File: package.json [hunk 2]
```diff
@@ -21,8 +21,8 @@
   "dependencies": {
     "@creativecommons/vocabulary": "1.0.0-beta.2",
     "@sentry/browser": "^5.6.0",
-    "@sentry/node": "^5.6.0",
-    "axios": "^0.18.0",
+    "@sentry/node": "^5.12.4",
```
Comment (akmadian) [inline package.json]: I'm not sure why package versions are being modified. Please revert all changes to package.json as dependency updates have the potential to be a separate issue.

File: src/components/NavSection.vue [hunk 3]
```diff
@@ -32,23 +35,31 @@
 
 <script>
 import { SET_QUERY } from '@/store/mutation-types';
+import HomeLicenseFilter from './HomeLicenseFilter';
 
 export default {
+  name: 'nav-section',
```
Comment (akmadian) [inline src/components/NavSection.vue]: This name does not match the component's name

File: src/components/NavSection.vue [hunk 4]
```diff
@@ -13,15 +13,18 @@
       <li class="nav_search" v-if="showNavSearch ==='true'">
         <form class="hero_search-form"
               role="search"
-              method="post"
+              method="get"
```
Comment (akmadian) [inline src/components/NavSection.vue]: Maybe I'm missing something, but why are changes to the markup necessary? All of the required elements are present on the page and this should mostly be behavioral changes.

Comment (akmadian) [reply src/components/NavSection.vue]: Everything looks pretty good (and works well) to me, but I'll defer to @brenoferreira, especially with the changes RE the method and action attributes.

File: src/components/NavSection.vue [hunk 5]
```diff
@@ -32,23 +35,31 @@
...
+    },
     navSearchPlaceholder: {
       default: 'Search all images',
     },
   },
-  name: 'nav-section',
   data: () => ({ form: { searchTerm: '' } }),
   methods: {
     onSubmit() {
+      // console.log(searchTerm);
```
Comment (akmadian) [inline src/components/NavSection.vue]: Instead of commenting, please just remove this

File: src/components/NavSection.vue (hunk 3 above)
Comment (abhisheknaiidu) [reply src/components/NavSection.vue]: We have NavSection as Component's name, earlier, it used to be the same name only, i just re-ordered the sequence.

**Comment by akmadian** [comment]
My apologies for not realizing this earlier and saving you time and work, but this PR essentially duplicates #642, and DantrazTrev was approved for work first, so I'm going to close this PR.

In general, please wait to be approved for work on an issue by a mentor or admin before working on a fix, and for more information, please see [CC's open source contribution guidelines](https://opensource.creativecommons.org/contributing-code/#contribution-process)

**Pull request closed without merging**